### ISSUE TITLE
Warnings when installing

### ISSUE THREAD

**Issue by balinterdi** [kiwiupover/ember-cli-surge#104]
When I install the latest version (1.1.21), I get the following, ember-sli-surge related warnings on the console:

```
warning ember-cli-surge > surge > tar.gz@0.1.1: ⚠️  WARNING ⚠️ tar.gz module has been deprecated and your application is vulnerable. Please use tar module instead: https://npmjs.com/tar
warning ember-cli-surge > surge > request > node-uuid@1.4.8: Use uuid module instead
```

The first especially seems important to fix as it's related to security.

I haven't checked but it's possible that the package has`surge` npm package dependency that is outdated?

```
$ ember -v
ember-cli: 2.16.2
node: 8.4.0
os: darwin x64
```

**Comment by kiwiupover** [comment]
Thanks, @balinterdi I will have a look.

**Comment by kiwiupover** [comment]
@balinterdi it is an upstream issue I will reach out to get the dep updated.

**Comment by balinterdi** [comment]
Wow, Dave, that was quick, thank you.

**Comment by kiwiupover** [comment]
@balinterdi you are the man I have to respond quickly.

**Comment by kiwiupover** [comment]
this should be closed by #124 

Thanks @balinterdi
